CHANGELOG.markdown

```diff
@@ -1,5 +1,11 @@
+5.1.5 [2020.xx.yy]
+------------------
+
+* Add `indexed-traversable` instances.
```

Comment:
It might be instructive to mention that these instances were migrated from `lens`. (The word "add" suggests that they are novel instances, which isn't really the case.)